functions.php

```diff
@@ -125,7 +125,7 @@ function twentysixteen_widgets_init() {
 	register_sidebar( array(
 		'name'          => esc_html__( 'Sidebar', 'twentysixteen' ),
 		'id'            => 'sidebar-1',
-		'description'   => '',
+		'description'   => __( 'Add widgets here to appear in your sidebar.', 'twentysixteen' ),
```

Comment:
I've updated the PR. I think [esc_html_2](https://codex.wordpress.org/Function_Reference/esc_html_2) is better option here, am I right?